x-pack/qa/multi-cluster-search-security/legacy-with-full-license/build.gradle

```diff
@@ -19,7 +19,7 @@ restResources {
 }
 
 // randomise between sniff and proxy modes
-boolean proxyMode = buildParams.random.nextBoolean()
```

Comment:
this resolves test seed early basically breaking configuration cache reusability every time